talk/oms/sdk/base/win/msdkvideobase.cc

```diff
@@ -40,22 +40,29 @@ bool MSDKFactory::Init() {
   return true;
 }
 
-MSDKFactory::~MSDKFactory() {
+MSDKFactory::~MSDKFactory() {}
+
+void MSDKFactory::SetMFETimeout(uint32_t timeout) {
```

Comment:
OMS C++ code usually use `type Property()` for getter and `void Property(type)` for setter.